types/coingecko-api/coingecko-api-tests.ts

```diff
@@ -39,7 +39,11 @@ const client = new CoinGecko();
         tickers: false
     })).data.market_data.total_volume.cad;
 
-    (await client.coins.fetchMarketChart("bitcoin", {vs_currency: "usd", days: "1"}));
```

Comment:
Might be worth keeping a copy of the test without interval to ensure that still works